example/fake-provider-cf.yml

```diff
@@ -160,7 +164,6 @@ Resources:
             local_enable=YES
             local_umask=022
             pam_service_name=vsftpd
-            tcp_wrappers=YES
```

Comment:
This is to fix the ftp response error:
curl -v ftp://user:pass@localhost/granules
< 500 OOPS: tcp_wrappers is set to YES but no tcp wrapper support compiled in
* Got a 500 ftp-server response when 220 was expected